tensorflow/python/util/deprecation_test.py

```diff
@@ -303,7 +303,7 @@ def _fn(self, arg0, arg1):
         "\n"
         "\nReturns:"
         "\n  Sum of args." % (date, instructions),
-        getattr(_Object, "_fn").__doc__)
+        _Object._fn.__doc__)
```

Comment:
every time you set/getattr for a private method, you need to add a comment at the end of the line:
```
... object._private ...  # pylint: disable=protected-access
```

otherwise the linter complains.

please do that at all locations.  also keep within the 80 char limit.

note we avoid using backslashes for code continuation, so instead of:

```
a = b + \
  c
```

use:
```
a = (b
     + c)

Reply:
👍 Fixed in 97db707